sno/output_util.py

```diff
@@ -0,0 +1,43 @@
+import io
+import json
+import sys
+
+
+def dump_json_output(output, output_path, pretty=True):
+    """
+    Dumps the output to JSON in the output file.
+    """
+
+    pretty = (not output_path) or output_path == "-" or output_path == sys.stdout
```

Comment:
You could just set pretty = (fp == sys.stdout) if you resolve the output path first.

Reply:
removed this line altogether; `pretty` is a kwarg so this was just overriding it